WordPress/openverse
# Auto-format markdown; switch to double quotes for JS

## Fixes


Fixes #333 by @dhruvkb
Fixes #287 by @sarayourfriend 

## Description


See my rationale for the approach here: https://github.com/WordPress/openverse/issues/333#issuecomment-1347478802

Notable, we want to have automated reflow of wrapped text so that writers can ignore it and write however they want. Having prose broken onto shorter lines allows more easily commenting on specific phrases when reviewing documents and discussions.

This also address the single quote issue with JavaScript linting. Naturally, this will create a massive diff in the frontend repository with the switch to double quotes.

I also fixed an issue with the `_js_install` recipe now that package.json has been moved to the top level of the repository.

## Testing Instructions

Checkout the branch, run `just install && just precommit && just lint` and verify that there are no changes left.

## Checklist

- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I added or updated documentation (if applicable).

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
